shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-route/src/main/java/org/apache/shardingsphere/sharding/route/engine/validator/util/StatementValidatorUtil.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.sharding.route.engine.validator.util;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.apache.shardingsphere.infra.exception.ShardingSphereException;
+import org.apache.shardingsphere.infra.metadata.ShardingSphereMetaData;
+import org.apache.shardingsphere.sharding.route.engine.exception.NoSuchTableException;
+import org.apache.shardingsphere.sharding.route.engine.exception.TableExistsException;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.generic.table.SimpleTableSegment;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * Statement validator utility class.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class StatementValidatorUtil {
```

Comment:
Is this class just for some of the unsupported DDLSQLStataments?  In that case, do you think an abstract class to carry these functions will be better?
`StatementValidatorUtil` gives me a  feeling that it is born for all the statements.

Reply:
> Is this class just for some of the unsupported DDLSQLStataments? In that case, do you think an abstract class to carry these functions will be better?
> `StatementValidatorUtil` gives me a feeling that it is born for all the statements.

@tristaZero I agree with your point, how about adding a new abstract class `ShardingDDLStatementValidator` to carry these functions？